File: 0009-palindrome-number/0009-palindrome-number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        if(x<0) return false;
        long long new_num = 0;
        long long old = x;
        while(x){
            new_num = new_num*10 + (x%10);
            x/=10;
        }
        if(old == new_num) return true;
        return false;
    }
};

File: 0009-palindrome-number/0009-palindrome-number.py
class Solution(object):
    def isPalindrome(self, x):
        if(x<0):
            return False
        s = ""
        while(x):
            s+=str(x%10)
            x=x//10
        i = 0
        j = len(s)-1
        while(i<=j):
            if(s[i]!=s[j]):
                return False
            i+=1
            j-=1
        return True